source/develop/release-management/features/network/unrestricted-network-names.html.md

```diff
@@ -43,17 +47,19 @@ We will make no change in the Engine/Vdsm API. This makes backward compatibility
 
 A new field, "vdsm_name", is to be added to the network entity. When a new network is added, its existing "name" has to be provided by the human actor. The new "vdsm_name" is to be auto-generated by Engine according to a pattern
 
-      brXXXXXXXXXXXXX
+      onXXXXXXXXXXXXX
 
-where `XXXXXXXXXXXXX` are 13 hexadecimal characters extracted from the UUID of the network. "name" is to be editable by the end user, and exposed as the name of the network over REST (as it is today); "vdsm_name" would be immutable, and to be used only for Engine/Vdsm communication.
+where `XXXXXXXXXXXXX` are 13 hexadecimal characters extracted from the UUID of the network. "name" is to be editable by the end user, and exposed as the name of the network over REST (as it is today); "vdsm_name" would be immutable once the network is attached to a host and to be used only for Engine/Vdsm communication.
```

Comment:
The creating of "vdsm_name" is slightly confusing.
Maybe replace (starting from "The new "vdsm_name" is to be auto-generated"):

The new "vdsm_name" will match "name", if "name" doesn't exceed IFNAMSIZ and only contains ASCII characters (to ease debugging and backwards compatibility).
Otherwise it will be auto-generated by Engine according to a pattern
	onXXXXXXXXXXXXX
where `XXXXXXXXXXXXX` are 13 hexadecimal characters extracted from the UUID of the network.
"name" is to be editable by the end user, and exposed as the name of the network over REST (as it is today).

In order to avoid any lingering states ......